generators/app/templates/src/config/ReactotronConfig.ejs

```diff
@@ -3,10 +3,15 @@ import Reactotron, { trackGlobalErrors } from 'reactotron-react-native';
 import apisaucePlugin from 'reactotron-apisauce';
 import { reactotronRedux } from 'reactotron-redux';
 
+// If you want to use a physical device and connect it to reactotron, execute first `adb reverse tcp:9090 tcp:9090`
+let scriptHostname;
 if (__DEV__) {
+  const scriptURL = NativeModules.SourceCode.scriptURL;
+  scriptHostname = scriptURL.split('://')[1].split(':')[0];
```

Comment:
do we always want this or only for ios?
Why is this needed?

Reply:
when i wanted to check redux status with QA's ios device, i couldn't get a connection with reactotron. I checked reactotron issues and got that ios uses the owner ip to connect with the device in this issue: https://github.com/infinitered/reactotron/issues/272
For android it works when using the comment above.